examples/pytorch/rgcn/entity_classify.py

```diff
@@ -59,14 +75,35 @@ def main(args):
     else:
         val_idx = train_idx
 
+    # calculate norm for each edge type and store in edge
+    for canonical_etypes in hg.canonical_etypes:
+        u, v, eid = hg.all_edges(form='all', etype=canonical_etypes)
+        _, inverse_index, count = torch.unique(v, return_inverse=True, return_counts=True)
+        degrees = count[inverse_index]
```

Comment:
Will it be easier to calculate degrees by `hg.in_degrees(canonical_etypes)`?

Reply:
Emm..
We can use in_degrees